node/src/accountant/db_big_integer/big_int_db_processor.rs

```diff
@@ -736,10 +837,10 @@ mod tests {
                 },
             },
             params_except_wei_change: vec![
-                Param::new(update_positive_2, true),
-                Param::new(update_negative_1, false),
-                Param::new(update_positive_1, true),
-                Param::new(update_negative_2, false),
+                Param::BothClauses(update_positive_2),
+                Param::MainClauseLimited(update_negative_1),
+                Param::BothClauses(update_positive_1),
+                Param::MainClauseLimited(update_negative_2),
             ],
         };
 
```

Comment:
I can't put this comment any lower than this, but this is related to the assertions below. I don't know why the result in such an order. In case there's a reason for that, maybe add a comment in the code.